django_ses/views.py

```diff
@@ -240,12 +247,20 @@ def handle_bounce(request):
                     },
                 )
 
-                signals.bounce_received.send(
-                    sender=handle_bounce,
-                    mail_obj=mail_obj,
-                    bounce_obj=bounce_obj,
-                    raw_message=raw_json,
-                )
+                if signal_sender == 'handle_bounce':
+                    signals.bounce_received.send(
+                        sender=handle_bounce,
+                        mail_obj=mail_obj,
+                        bounce_obj=bounce_obj,
+                        raw_message=raw_json,
+                    )
+                else:
```

Comment:
Just wondering, why is this split necessary?

Reply:
I didn't want to change `sender` parameter of receiver.  I have added if for the users who checks `sender`. I may not be necessary. I have never check `sender` method in receiver.